ebpf_asm.py

```diff
@@ -739,7 +744,7 @@ def assemble_alu(self, insn):
     def assemble_jmp(self, insn):
         # class, op, dest, {x, src | k, imm}, off
         off = insn.get('off', 0)
-        if isinstance(off, (int, long)):
+        if isinstance(off, (int, int)):
```

Comment:
Presumably we don't need `int` twice here (guessing this is the autoconverter's doing).
Just `if isinstance(off, int):` seems like the sensible thing.

I've had a quick dig through and I don't _think_ this can ever be a `long` anyway even in py2; but it worries me a little.  Since the alternative possibility for `off` here is a label name, maybe this should instead be
`if not isinstance(off, str):`
just to be on the safe side.

Reply:
Your guess would be correct! I think your suggestion is very reasonable, and have changed the code accordingly.